packages/twenty-front/src/modules/ui/utilities/drag-select/components/DragSelect.tsx

```diff
@@ -43,11 +51,21 @@ export const DragSelect = ({
     onSelectionStart: onDragSelectionStart,
     onSelectionEnd: onDragSelectionEnd,
     onSelectionChange: (box) => {
+      const containerTopOnScreen =
```

Comment:
you can leverage useScrollToPosition + scrollWrapper states to know scroll positions. You should be able to handle the autoScroll with a small amount of code.

What's left:
- this example only handle scrollTop when the selection is going up (missing 3 other cases you had in your PR)
- we should handle the case when we have no scrollWrapper (just default to 0 I guess)